1.fee_bumping/fee_bumping.md

```diff
@@ -153,25 +153,208 @@ Bitcoin blockchain, such as exchanges or custodians:
 
 These issues will be discussed in more detail in later sections of this document.
 
-# Introduction to RBF
-
-TODO
-
-## Overview of what RBF is
-
-TODO
-
-## History of RBF
-
-TODO
+# Transaction replacement
+
+The concept of being able to 'replace' an unconfirmed transaction has been
+around since the very earliest days of Bitcoin. Early versions of the
+Bitcoin reference implementation [allowed transactions to be replaced][full_replacement] by
+constructing and broadcasting a replacement transaction with modified
+`nSequence` numbers in the transaction inputs.
+
+Satoshi understood the need for transaction fees to increase as the subsidy
+decreased, saying in the [whitepaper][] 'Once a predetermined number of coins
+have entered circulation, the incentive can transition entirely to transaction
+fees and be completely inflation free.' He also understood that users would
+need a way to be able to bump the fee on a transaction if it got stuck. There
+was [a question on the bitcointalk forum][bct q] in 2010:
+
+> Just wondering about the following example :
+> I broadcast a transaction, sending X coins to some address.
+> Doesn't get included in blocks for a while because I don't include a fee.
+> 
+> Do I have a way to cancel it and broadcast it again with a fee this time ?
+
+Satoshi [replied][bct a]:
+
+> There's a possible design for far in the future:
+> 
+> You intentionally write a double-spend.  You write it with the same inputs
+> and outputs, but this time with a fee.  When your double-spend gets into a
+> block, the first spend becomes invalid.  [...]
+> 
+> It's easier said than implemented.  There would be a fair amount of work[...]
+
+There are two main areas of difficulty that Satoshi is talking about: wallet
+design and UX, and node and miner policy.
+
+The wallet design and UX questions (how should a wallet track and display
+conflicting transactions? How should it present a user interface for bumping
+fee?) can be resolved by individual wallets and applications.
+
+The node an miner policy difficulties require some kind of shared
```

Comment:
@dgpv says ""The node **an** miner policy difficulties require some kind of shared understanding"
should be "and"